### PR TITLE
[ADD] <SuperDesigner.Co>

### LINKED ISSUE #883: [Add] <SuperDesigner.Co>
### Site URL

https://superdesigner.co/

### Description of the above provided site

This site has all types of Generators such as 

1. Svg backgrounds
2. 3d shapes generator
3. Background generator
4. Pattern generator
5. Blob generator
6. Gradient generator

Best thing is that its all free and you don't need to login

### Category

Frontend

### Sub Category

UI generator

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by PriyansuMaurya**
## Fixes Issue

[Add] <SuperDesigner.Co> Closes #883

## Screenshots

image

### CODE REVIEW

File: database/frontend/ui-generators.json [hunk 1]
```diff
@@ -117,5 +117,12 @@
     "url": "https://css-grid-generator.netlify.app/",
     "category": "frontend",
     "subcategory": "ui-generators"
+  },
+  {
+    "name": "SuperDesigner.Co",
+    "description": "Generate stunning design elements effortlessly with this free website that offers a diverse range of generators, including SVG backgrounds, 3D shapes, patterns, blobs, and gradients - no login required!.",
```
Comment (rupali-codes) [inline database/frontend/ui-generators.json:123]:
```suggestion
    "description": "Generate stunning design elements effortlessly with this free website that offers a diverse range of generators, including SVG backgrounds, 3D shapes, patterns, blobs, and gradients.",
```

**Comment by PriyansuMaurya** [comment]
made the requested changes

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by ujjawaltyagii — APPROVED** [review]
lgtm !!

**Pull request merged**